src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Extensions/ConfigurationBuilderExtensions.cs

```diff
@@ -1,16 +1,21 @@
-using System.IO;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.FileProviders;
 
 namespace Altinn.App.PlatformServices.Extensions
 {
     public static class ConfigurationBuilderExtensions
     {
-        public static void LoadAppConfig(this IConfigurationBuilder builder)
+        public static void LoadAppConfig(this IConfigurationBuilder builder, string[] args = null)
         {
-            builder.AddJsonFile(Directory.GetCurrentDirectory() + @"/appsettings.json", true, true);
+            builder.AddJsonFile(
+                new PhysicalFileProvider("/"),
+                @"altinn-appsettings-secret/altinn-appsettings-secret.json",
+                true,
+                true);
+
+            // Add values from environment and command line arguments last, to override values from other sources.
             builder.AddEnvironmentVariables();
```

Comment:
are we sure we want them to be able to override the altinn-appsettings-secret ?

Reply:
I don't see any issues with that at the moment.